webserver-windows.ps1

```diff
@@ -50,3 +50,15 @@ else {
     multipass launch lts --name relativepath
 }
 
+
+# Create ssh key pair if it does not exist for a given user
+# gets env username : [Environment]::UserName
+# gets user's home path: $env:USERPROFILE
+$userName = [Environment]::UserName
```

Comment:
Why are you setting a variable here? Could you just do this inline in the conditional?

Reply:
I'm setting the variable above the conditional as it's called in the if and else clause (and also the cloud-init yaml file that I'm working on). 

I'm also unfamiliar/uncertain of the scope of this variable but I'm hopeful that it will be available from the yaml file anyway.